### ISSUE TITLE
Knockback strange

### ISSUE THREAD

**Issue by Clashbestie** [MeteorDevelopment/meteor-client#142]
Knockback is acting very strange, as seen in this video (I only pressed space and w), playing on a Paper-Server

https://user-images.githubusercontent.com/57112209/104510645-c07c9e00-55eb-11eb-9f0c-2f0c61e82b5a.mp4

**Comment by seasnail8169** [comment]
You clearly only have the HUD active therefore its clearly not meteor that is causing whatever issue you are having. A module would have to be active for meteor to cause something remotely similar to that.

**Comment by Clashbestie** [comment]
Relax, xD.
I did some testing:
Yeah you are right I only have the HUD active but Velocity still kinda manipulates the Velocity applied.
Why? Don't ask me the code seems fine, but I know [where ](https://github.com/MeteorDevelopment/meteor-client/blob/252a72910dab2ac335cf164f78fe72502dcf71b4/src/main/java/minegame159/meteorclient/mixin/EntityMixin.java#L53).
Adding `if (velocity.isActive())` infront of the line resolves this problem.
Idk why the returned values from Velocity have a effect when disabled, because y*1 is still y?
I logged the values, they are still the same after multipling, but the y(the one thats provided as arg to the funtion) spikes sometimes, idk why.

And btw the client is awesome.
